data/attributes.yml

```diff
@@ -30515,13 +30515,13 @@ base_attributes:
 - id: 3297
   name: Humidy control
```

Comment:
We also want to change this to `Humidity control method`, right? I think that's what's causing the integration tests to fail.